Fixed error that triggered error dialog in Display Daily dialog

EstherNjeriLiberatta opened the pull request:
Fixes  #7016 

@N-thony kindly review this. I think the error is now fixed.

N-thony commented on instat/dlgDisplayDailyData.vb:
```diff
@@ -240,7 +240,7 @@ Public Class dlgDisplayDailyData
 
 
-        ucrChkMissing.AddToLinkedControls({ucrInputComboMissing}, {True}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
+        ucrChkMissing.AddToLinkedControls({ucrInputComboMissing}, {True}, bNewLinkedUpdateFunction:=True, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
```
Do you need bNewLinkedUpdateFunction:=True?

EstherNjeriLiberatta replied:
Hello @N-thony I've commented out the SetRCode for ucrInputComboMissing and replaced it with that. It does the same thing but it helps remove the confusion between the `ucrChkMissing` and `ucrInputComboMissing` controls.

N-thony commented on instat/dlgDisplayDailyData.vb:
```diff
@@ -177,7 +177,7 @@ Public Class dlgDisplayDailyData
         lstCheckboxes.AddRange({ucrChkSum, ucrChkMax, ucrChkMin, ucrChkMean, ucrChkMedian, ucrChkIQR, ucrChkSumMissing})
 
-        ucrChkMissing.SetParameter(ucrInputComboMissing.GetParameter)
+        ' ucrChkMissing.SetParameter(ucrInputComboMissing.GetParameter)
```
Remove? Same with other commented line

N-thony reviewed (changes requested):
Under Table option. Can you fix this small design issue with missing checkbox going over its groupbox
![image](https://user-images.githubusercontent.com/68591383/145191414-977ade0f-9936-4202-918c-a73199a9274c.png)

rdstern reviewed (dismissed):
seems ok now

EstherNjeriLiberatta commented:
@N-thony I fixed the checkbox overlap. I also noticed an inconsistency in spelling of colour on the `Rug color` label. I fixed that too. Thanks.

@rdstern This is ready for review.

lloyddewit commented:
> @N-thony I fixed the changes you requested. 

@EstherNjeriLiberatta I don't think you resolved @N-thony 's comments in `dlgDisplayDailyData.vb`, please could you double-check? thanks

EstherNjeriLiberatta commented:
@lloyddewit No problem, thank you.

EstherNjeriLiberatta commented:
Hello @N-thony please review the changes I've made.
@rdstern @africanmathsinitiative/developers This is ready for review.

rdstern reviewed (approved):
Great.
